specs/DotNetProjectFile.Analyzers.Specs/Design_specs.cs

```diff
@@ -30,7 +32,9 @@ public void Root_Readme_mentions_right_number_of_rules()
         var amount = int.Parse(match.Groups["amount"].Value);
 
         var all = AlleRules.Count();
-
+#if DEBUG
+        Console.WriteLine($"Contains {all} rules.");
```

Comment:
Intentional?

Reply:
It is the fastest way to check the number of rules we currently have. (115 <3)